app/src/main/java/com/example/dima_2/lesson5/ItemsActivity.java

```diff
@@ -0,0 +1,72 @@
+package com.example.dima_2.lesson5;
+
+import android.app.ListActivity;
+import android.app.LoaderManager;
+import android.content.CursorLoader;
+import android.content.Intent;
+import android.content.Loader;
+import android.database.Cursor;
+import android.net.Uri;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.ListView;
+import android.widget.SimpleCursorAdapter;
+
+/**
+ * Created by Dima_2 on 16.01.2015.
+ */
+public class ItemsActivity extends ListActivity implements LoaderManager.LoaderCallbacks<Cursor> {
+
+    private SimpleCursorAdapter adapter = null;
+    private int channelID;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        channelID = getIntent().getIntExtra("channel_id", 0);
+        String[] f = new String[]{ItemsTable.COLUMN_TITLE, ItemsTable.COLUMN_DESCRIPTION};
+        int[] t = new int[] {android.R.id.text1, android.R.id.text2};
+        adapter = new SimpleCursorAdapter(this, android.R.layout.simple_list_item_2, null, f, t, 0);
+        setListAdapter(adapter);
+        getLoaderManager().initLoader(0, null, this);
+    }
+
+    private void fillData() {
+        getLoaderManager().initLoader(0, null, this);
+    }
+
+    @Override
+    protected void onListItemClick(ListView listView, View view, int i, long id) {
+        super.onListItemClick(listView, view, i, id);
+        Uri elementsUri = Uri.withAppendedPath(DatabaseContentProvider.CONTENT_URI_ITEMS, String.valueOf(id));

```

Comment:
All I wanted - to get cursor and then do some operations with it. This way is not good, but I needed anything that works and it works :)